crowdin_api/api_resources/projects/resource.py

```diff
@@ -372,3 +375,114 @@ def edit_project_file(
             ),
             request_data=data,
         )
+
+    def get_strings_exporter_path(
+        self,
+        projectId: int,
+        systemStringExporterSettingId: Optional[int] = None
+    ):
+        if systemStringExporterSettingId is None:
+            return f"projects/{projectId}/strings-exporter-settings"
+        return f"projects/{projectId}/strings-exporter-settings/{systemStringExporterSettingId}"
+
+    def list_project_strings_exporter(
+        self,
+        projectId: int,
+        page: Optional[int] = None,
+        offset: Optional[int] = None,
+        limit: Optional[int] = None,
```

Comment:
this method doesn't accept such parameters

Reply:
For this I looked at `list_project_files` and  there are params like this and response for both are same
https://github.com/crowdin/crowdin-api-client-python/blob/3b598b9a6795b36de3bfea43d40fdee77d266938/crowdin_api/api_resources/projects/resource.py#L278-L284

Reply:
@OmAximani0 please remove